src/main/java/com/nike/cerberus/operation/core/CreateBaseOperation.java

```diff
@@ -69,10 +72,22 @@ public void run(final CreateBaseCommand command) {
 
         final Map<String, String> parameters = cloudFormationObjectMapper.convertValue(baseParameters, typeReference);
 
-        cloudFormationService.createStack(Stack.BASE, parameters, true, command.getTagsDelegate().getTags());
-
-        configStore.initEnvironmentData();
-        configStore.initSecretsData();
+        final String stackId = cloudFormationService.createStack(Stack.BASE, parameters, true,
+                command.getTagsDelegate().getTags());
+
+        final StackStatus endStatus =
```

Comment:
nit: I think it would be a nice refactor if the "wait for status" logic went into it's own method. Maybe even if there was a `createStackAndWaitForStatus` method

Reply:
That would probably work, I went with the pattern that was already in use in the CLI.

Reply:
I think we should definitely do one or the other.

Looks good, otherwise. Approved